core/src/test/java/com/linecorp/armeria/server/RoutingPredicateTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 LINE Corporation
+ * Copyright 2022 LINE Corporation
```

Comment:
No need to update the inception year when modifying an existing file.
```suggestion
 * Copyright 2019 LINE Corporation
```